### PR TITLE
Use local loopback address instead of 'any' address for ZkImplTest

### LINKED ISSUE #1197: ZkImplTest fails on MacOS
ZkImplTest times out.  Connecting to 0.0.0.0 ip address does not work on BSD as it does on Linux.

### PR DESCRIPTION
**Pull request by chonton**
closes #1197

### Type of change
- Bugfix

### Description
Use local loopback address instead of 'any' address for ZkImplTest

### Checklist

_Please go through this checklist and make sure all applicable tasks have been done_

- [N/A] Update/write design documentation in `./design`
- [N/A] Write tests
- [Done] Make sure all tests pass
- [N/A] Update documentation
- [N/A] Check RBAC rights for Kubernetes / OpenShift roles
- [N/A] Try your changes from Pod inside your Kubernetes and OpenShift cluster, not just locally
- [N/A] Reference relevant issue(s) and close them after merging
- [N/A] Update CHANGELOG.md

### CODE REVIEW

**Comment by chonton** [comment]
At home, the any interface works.  At work, it does not.  Maybe something to do with the routing tables?

File: test/src/main/java/io/strimzi/test/EmbeddedZooKeeper.java [hunk 1]
```diff
@@ -21,7 +22,7 @@
     public EmbeddedZooKeeper() throws IOException, InterruptedException {
         dir = Files.createTempDirectory("strimzi").toFile();
         zk = new ZooKeeperServer(dir, dir, 1000);
-        start(new InetSocketAddress(0));
+        start(new InetSocketAddress(Inet4Address.getLoopbackAddress(), 0));
```
Comment (tombentley) [inline test/src/main/java/io/strimzi/test/EmbeddedZooKeeper.java]:
```suggestion
        start(new InetSocketAddress(InetAddress.getLoopbackAddress(), 0));
```

File: test/src/main/java/io/strimzi/test/EmbeddedZooKeeper.java [hunk 2]
```diff
@@ -9,6 +9,7 @@
 
 import java.io.File;
 import java.io.IOException;
+import java.net.Inet4Address;
```
Comment (tombentley) [inline test/src/main/java/io/strimzi/test/EmbeddedZooKeeper.java]:
```suggestion
import java.net.InetAddress;
```

**Comment by tombentley** [comment]
Thanks!

**Pull request merged**